addons/flannel/0.21.5/install.sh

```diff
@@ -233,29 +233,42 @@ function weave_to_flannel() {
     restart_systemd_and_wait kubelet
 
     logStep "Restarting pods in kube-system"
-    kubectl -n kube-system delete pods --all --grace-period=200
-    kubectl -n kube-flannel delete pods --all --grace-period=200
+    log "this may take several minutes"
+    local ns=
+    namespaces=("kube-system" "kube-flannel")
+
+    for ns in "${namespaces[@]}"; do
+        log "Waiting up to 5 minutes to delete all pods from ${ns}"
+        if ! spinner_until 300 kubectl -n ${ns} delete pods --all --ignore-not-found; then
```

Comment:
it is required that these pods be deleted to get new pod sandboxes. i do not think a timeout with a warning and proceeding is a good idea.